kie-server-parent/kie-server-remote/kie-server-client/src/build/revapi-config.json

```diff
@@ -95,8 +93,20 @@
               "methodName": "suspendTask",
               "elementKind": "method",
               "justification": "[JBPM-10015] Allow to set suspendUntil during user task suspend operation as parameter."
+            },
+            {
```

Comment:
@bxf12315 this block is the only change is needed here (regarding numberOfParametersChanged)